integration-tests/Cargo.toml

```diff
@@ -26,3 +43,9 @@ protocol_feature_alt_bn128 = ["nearcore/protocol_feature_alt_bn128"]
 nightly_protocol_features = ["nearcore/nightly_protocol_features", "testlib/nightly_protocol_features", "protocol_feature_alt_bn128", "protocol_feature_restore_receipts_after_fix"]
 nightly_protocol = ["nearcore/nightly_protocol", "testlib/nightly_protocol"]
 protocol_feature_restore_receipts_after_fix = ["nearcore/protocol_feature_restore_receipts_after_fix"]
+sandbox = ["near-network/sandbox", "near-chain/sandbox", "node-runtime/sandbox", "near-client/sandbox"]
+
+[[test]]
+name = "sandbox"
+required-features = ["sandbox"]
+path = "tests/client/sandbox.rs"
```

Comment:
I think this doesn't do what you want this todo. As written, the `tests/client/sandbox.rs` file is part of **two** test crates.
One has the root at `tests/client/sandbox.rs`, the other at `tests/client/main.rs`. 

I think you can completely remove the `[[test]]` section if you gate `mod test;` by the cfg as suggested in other ccomment.

Reply:
Done